### ISSUE TITLE
Server-side enums (used in endpoints) are broken / prevent app startup in 19.0.7

### ISSUE THREAD

**Issue by markhm** [vaadin/fusion#6]
### Description of the bug 
**Major Blocker:** After upgrading to 19.0.7 from 19.0.5, our Fusion application that uses server-side `enum`s no longer starts with the following error.

```
WARNING in ./views/xx/yy/zz/qqqq-view.ts 112:27-37
"export 'default' (imported as 'SyncStatus') was not found in 'generated/aa/bb/cc/SyncStatus'
 @ ./routes/routes-main.ts
 @ ./routes/routes-all.ts
 @ ./index.ts
 @ ./generated/vaadin.ts
```

There are dozens more of such errors, all for / for all the enums in our application. 

### Minimal reproducible example
Create a simple server-side enum and use this enum in an endpoint, e.g. 

```
public enum SyncStatus {
    REQUESTED,
    PROCESSING,
    ERRORED,
    FINISHED
}
```

### Expected behavior
Enums should work normally as they did in 19.0.5. 

### Actual behavior
The following TS is generated for 19.0.7:

```
const enum SyncStatus {
  REQUESTED = 'REQUESTED',
  PROCESSING = 'PROCESSING',
  ERRORED = 'ERRORED',
  FINISHED = 'FINISHED',
}
export default SyncStatus;
```

The TS code that is generated for 19.0.5 seems identical. 

### Versions:
    - Vaadin / Fusion version: 19.0.7 (upgraded from 19.0.5)
    - Node version: v16.1.0
    - Java version: 16
    - OS version: macOS latest

**Comment by markhm** [comment]
These are some of the changes I could see in `package.json`:

- "lit-element": "2.3.1",
+ "lit-element": "2.5.1",
- "lit-html": "1.3.0",
+ "lit-html": "1.4.1"

Hope this helps. Thanks for looking into this. 🙏

**Comment by haijian-vaadin** [comment]
Hi @markhm, thanks for the feedback. I check the commit history, seems we didn't have enum-related changes.  I tried to reproduce the problem with a simple starter project + Endpoint with enum, but seems it works fine for me. 
Can you help check this [example](https://github.com/haijian-vaadin/v19-enum-test) (HelloEndpoint) and let us know if you are using the enum in a different way?

**Comment by haijian-vaadin** [comment]
Figured the reason was that the [ts-loader change](https://github.com/vaadin/flow/commit/646d9a8189aac986ac98c94324ab55486546246c) affected the enum generation, the [use regular enum instead of const enum change](https://github.com/vaadin/flow/pull/10853) should fix this problem (which is delivered in Flow 6.0.9, to be included in 19.0.8). 

So close this issue for now, and let's reopen it if it's still an issue.
